dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/AbstractHTTPDestination.java

```diff
@@ -702,11 +712,11 @@ private int getReponseCodeFromMessage(Message message) {
         Integer i = (Integer) message.get(Message.RESPONSE_CODE);
         if (i != null) {
             return i.intValue();
+        } else if (hasNoResponseContent(message)) {
+            return HttpURLConnection.HTTP_ACCEPTED;
+        } else {
+            return HttpURLConnection.HTTP_OK;
         }
-        int code = hasNoResponseContent(message) ? HttpURLConnection.HTTP_ACCEPTED : HttpURLConnection.HTTP_OK;
-        // put the code in the message so that others can get it
-        message.put(Message.RESPONSE_CODE, code);
```

Comment:
I don't like that we are no longer putting the Response Code in the message with you change. I think it should be reverted back to the original